singleheader/CMakeLists.txt

```diff
@@ -94,5 +94,5 @@ add_dependencies(simdjson-singleheader-source amalgamate)
 # Test the generated simdjson.cpp/simdjson.h using the generated amalgamate_demo.cpp
 #
 add_executable(amalgamate_demo $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/amalgamate_demo.cpp>)
-target_link_libraries(amalgamate_demo simdjson-singleheader-include-source simdjson-flags)
+target_link_libraries(amalgamate_demo simdjson-singleheader-include-source simdjson-internal-flags simdjson-flags)
```

Comment:
I think we should have: `target_link_libraries(simdjson-internal-flags simdjson-flags)` so we don't have to remember to do this :)